### PR TITLE
explicitly describe apt-get results

### PR DESCRIPTION
**Pull request by westonplatter**

### CODE REVIEW

**Comment by westonplatter** [comment]
@zzet I keep getting install issues with the apt_cache.

``` sh
failed: [example.com] => {"failed": true, "item": ""}
msg: Failed to lock apt for exclusive operation

FATAL: all hosts have already failed -- aborting
```

I moved the apt_cache command within the install process and then only update when apt_cache is less than 1 hour old. 

How does this look to you @zzet ?

File: tasks/main.yml [hunk 1]
```diff
@@ -12,17 +12,13 @@
...
   command: host -t A ansible.cc
 
-- name: make sure the system encoding is {{ locale }}
+- name: "make sure the system encoding is {{ locale }}"
   sudo: yes
   template: src=locale dest=/etc/default/locale owner=root group=root mode=644
 
-- name: ensure apt cache is up to date
-  sudo: yes
-  apt: update_cache=yes
```
Comment (zzet) [inline tasks/main.yml:21]: @westonplatter I read practices that say it's better to share the installation with caching. I think that in our case it is not critical. If there are problems - the rollback.

Comment (westonplatter) [inline tasks/main.yml:21]: Ok. Maybe we should not remove "ensure apt cache is up to date".

Have you had the same error? msg: Failed to lock apt for exclusive operation

Comment (zzet) [inline tasks/main.yml:21]: @westonplatter Hm... do you have this error while install role to clean box?

**Pull request merged**